### DISCUSSION TITLE
Not able to embed superset dashboard in iframe

### DISCUSSION THREAD

**Discussion by shafad1** [apache/superset#27443] [General]
I am trying to embed superset dashboard in another application GUI by referring haproxy. for authentication using keyclock(openid) .    
getting below error 
 **Blocked autofocusing on a <input> element in a cross-origin subframe.**

#This is my proxy configuration .(proxy ip 10.0.10.9:8080, Keyclock server=10.0.10.51:8180)
 frontend Ooredoo_OM_DMS
 bind *:8080
 bind-process 1 2 3 4 5 6 7 8 9 10 11 12
 option tcplog
 mode http
 #redirect scheme https code 301 if !{ ssl_fc }X-
 acl is_dmsbl path -i -m beg /auth
 use_backend keyclock if is_dmsbl
 acl is_ui  path  -i -m beg  /baseui
 use_backend iframe  if is_ui
 default_backend superset

error:
![image](https://github.com/apache/superset/assets/107529872/40d5bc20-6fd4-42e2-b274-6a142e396062)

Can anyone please help me on this

**Comment by Keveen-ghori** [comment]
I am facing same issue. Have you solved it?
